sessionctx/variable/session.go

```diff
@@ -801,3 +809,49 @@ type BatchSize struct {
 	// MaxChunkSize defines max row count of a Chunk during query execution.
 	MaxChunkSize int
 }
+
+const (
+	SlowLogPrefixStr     = "# "
+	SlowLogSpaceMarkStr  = ": "
+	SlowLogSQLSuffixStr  = ";"
+	SlowLogTimeStr       = "Time"
+	SlowLogTxnStartTSStr = "Txn_start_ts"
+	SlowLogUserStr       = "User"
+	SlowLogConnIDStr     = "Conn_ID"
+	SlowLogQueryTimeStr  = "Query_time"
+	SlowLogDBStr         = "DB"
+	SlowLogIsInternalStr = "Is_internal"
+	SlowLogIndexNamesStr = "Index_names"
+	SlowLogQuerySQLStr   = "Query" // use for slow log table, slow log will not print this field name but print sql directly.
+)
+
+// SlowLogFormat uses for formatting slow log.
+func (s *SessionVars) SlowLogFormat(txnTS uint64, costTime time.Duration, execDetail execdetails.ExecDetails, indexNames string, sql string) string {
+	var buf bytes.Buffer
+	execDetailStr := execDetail.String()
+	buf.WriteString("# " + SlowLogTxnStartTSStr + ": " + strconv.FormatUint(txnTS, 10) + "\n")
```

Comment:
why you don't use `SlowLogPrefixStr` to replace `#` here?

Reply:
done.